package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "test": "npm run lint && npm run unit && npm run tsd",
     "lint": "standard",
-    "unit": "tap test/test.js test/*.test.js --100",
+    "unit": "tap test/test.js test/*.test.js",
```

Comment:
Why did the coverage go down?

Reply:
The `validate.js` file is auto-generated and it has a lower coverage threshold. Note that I just moved the `--100` option to `.taprc`.